lldb/include/lldb/Core/PluginManager.h

```diff
@@ -610,6 +610,14 @@ class PluginManager {
   static bool CreateSettingForCPlusPlusLanguagePlugin(
       Debugger &debugger, const lldb::OptionValuePropertiesSP &properties_sp,
       llvm::StringRef description, bool is_global_property);
+
+  static lldb::OptionValuePropertiesSP
+  GetSettingForSwiftLanguagePlugin(Debugger &debugger,
```

Comment:
Again, it would be nice if this could live in Plugin/ (but maybe it's harder to pull off here?)

Reply:
These static methods are only called in `SwiftLanguage.cpp`. They could be moved to that file, but that would break the convention of the `PluginManager`. I think it would be better that these methods live in their respective `XLanguage.cpp` file, to keep language specific components separated. What do you think @Michael137 ?